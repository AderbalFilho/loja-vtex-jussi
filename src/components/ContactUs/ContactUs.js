import React from 'react';

import { JLink, JText, JTitle, JSection } from '@ui';

import './ContactUs.scss';

function ContactUs() {
  return (
    <JSection color="pink" classes="contact-us">
      <div className="contact-us__container">
        <JTitle tag="h2" color="white" classes="contact-us__title">
          Essa loja foi construída usando uma das nossas soluções da plataforma
          VTEX. tenha a sua.
        </JTitle>
        <JText color="white" classes="contact-us__contact">
          Entre em contato
        </JText>
        <JLink href="mailto:comercial@jussi.com.br" classes="contact-us__mail">
          comercial@jussi.com.br
        </JLink>
      </div>
    </JSection>
  );
}

export default ContactUs;
