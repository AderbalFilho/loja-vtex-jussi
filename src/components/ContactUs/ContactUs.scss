.contact-us {
  padding: 136px 0;
}

.contact-us__container {
  margin: 0 auto;
  max-width: 676px;
  text-align: center;
  width: 100%;
}

.contact-us__title {
  margin: 0 0 32px 0;
}

.contact-us__contact {
  margin: 0 0 8px 0;
}

.contact-us__mail {
  color: var(--white);
  font-size: 1.5rem;
  line-height: 1rem;

  @media (min-width: 375px) {
    font-size: 2rem;
    line-height: 1.5rem;
  }
}
