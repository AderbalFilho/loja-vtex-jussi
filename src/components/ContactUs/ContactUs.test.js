import React from 'react';
import TestRenderer from 'react-test-renderer';

import ContactUs from './ContactUs';

describe('ContactUs', () => {
  it('matches snapshot', () => {
    const testRenderer = TestRenderer.create(<ContactUs />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
