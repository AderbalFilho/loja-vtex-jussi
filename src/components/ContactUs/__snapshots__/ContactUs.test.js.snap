// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs matches snapshot 1`] = `
<section
  className="j-section j-section--pink contact-us"
>
  <div
    className="contact-us__container"
  >
    <h2
      className="j-title j-title--white j-title--large contact-us__title"
    >
      Essa loja foi construída usando uma das nossas soluções da plataforma VTEX. tenha a sua.
    </h2>
    <p
      className="j-text j-text--white j-text--normal contact-us__contact"
    >
      Entre em contato
    </p>
    <a
      className="j-link contact-us__mail"
      href="mailto:comercial@jussi.com.br"
    >
      comercial@jussi.com.br
    </a>
  </div>
</section>
`;
